### PR TITLE
[WIP] Port to Kirigami 2

### LINKED ISSUE #80: Port to Kirigami2

### PR DESCRIPTION
**Pull request by JBBgameich**
- [x] Base works with Kirigami 2
- [ ] Controls are shown (This is strange and I haven't yet figured out while controls aren't shown after login again)

Please do not only check the code, but test build and use it, so I can see what has been changed in kirigami 2

Closes #80

### CODE REVIEW

File: src/qml/RosterPage.qml [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
+import org.kde.kirigami 2.0 as Kirigami
 
 Kirigami.ScrollablePage {
 	title: qsTr("Contacts")
 
 	ListView {
 		model: kaidan.rosterController.rosterModel
 
-		delegate: Kirigami.BasicListItem {  //Kirigami.SwipeListItem
+		delegate: Kirigami.AbstractListItem {  //Kirigami.SwipeListItem
```
Comment (LNJ2) [inline src/qml/RosterPage.qml]: Nope. SwipeListItem

Comment (JBBgameich) [inline src/qml/RosterPage.qml]: Yeay, right, it's ok to use it on this branch even if there is no function. I will change that soon

Comment (LNJ2) [inline src/qml/RosterPage.qml]: :+1:

File: src/qml/RosterPage.qml [hunk 2]
```diff
@@ -42,14 +45,14 @@ Kirigami.ScrollablePage {
...
-			// 		onTriggered: {
-			// 			//
-			// 		}
-			// 	}
-			// ]
+			actions: [
+			 	Kirigami.Action {
+			 		iconName: "bookmark-remove"
+			 		onTriggered: {
+
```
Comment (LNJ2) [inline src/qml/RosterPage.qml]: NO SPACES!

Comment (JBBgameich) [inline src/qml/RosterPage.qml]: Oh! Haven't noticed I put spaces there ... Will correct

**Comment by JBBgameich** [comment]
Yeay, there are only the controls still not shown, that means the drawer is missing it's open-close button and there are no headings for the pages shown.

File: src/qml/RosterPage.qml [hunk 3]
```diff
@@ -28,10 +28,9 @@ Kirigami.ScrollablePage {
 		model: kaidan.rosterController.rosterModel
 
 		delegate: Kirigami.SwipeListItem {
-			height: contactText.height + 16
+			height: 50
```
Comment (LNJ2) [inline src/qml/RosterPage.qml]: Better use a `Kirigami.Units.gridUnit` (see https://phabricator.kde.org/source/kirigami/browse/master/src/controls/Units.qml;a90b442d06b3b64dd681c84dcc7aa8c61fa53a29$35)

File: src/qml/RosterPage.qml [hunk 4]
```diff
@@ -42,14 +44,13 @@ Kirigami.ScrollablePage {
...
-			// 		}
-			// 	}
-			// ]
+			actions: [
+			 	Kirigami.Action {
+			 		iconName: "bookmark-remove"
+			 		onTriggered: {
+					}
+				}
+			]
```
Comment (LNJ2) [inline src/qml/RosterPage.qml]: Generally this is completely different thing and has nothing to do with Kg2, but it's ok.

**Pull request merged**